mapsforgesrv/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -1,9 +1,9 @@
 eclipse.preferences.version=1
 org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
 org.eclipse.jdt.core.compiler.codegen.methodParameters=do not generate
-org.eclipse.jdt.core.compiler.codegen.targetPlatform=11
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=17

```

Comment:
target platform should be 11. i think, otherwise the builded jar file is not running with java >17
correct?

Reply:
Correct ! it must be 11